### ISSUE TITLE
Show Parameters doesn't work without valid Security Scoped Bookmark

### ISSUE THREAD

**Issue by latenitefilms** [latenitefilms/BRAWToolbox#111]
As discussed in #101.

Untested, but basically I think if you're relying on **Grant Sandbox Access** to access a BRAW file, the **Show Parameters** won't work.

**Comment by latenitefilms** [comment]
Confirmed - I get this error:

```
2023-04-09 07:27:34.905 BRAW Toolbox Renderer[58428:6237633] [BRAW Toolbox Renderer] ERROR: populateBRAWParametersWithReset Bookmark Error: The file couldn’t be opened.
2023-04-09 07:27:34.906 BRAW Toolbox Renderer[58428:6237633] [BRAW Toolbox Renderer] FATAL ERROR: Failed to open IBlackmagicRawClip in parameterChanged.
```

**Comment by latenitefilms** [comment]
This will be fixed in v1.0.7.
